joern-cli/src/main/scala/io/shiftleft/joern/Cpg2Scpg.scala

```diff
@@ -21,24 +23,34 @@ object Cpg2Scpg extends App {
     System.exit(0)
   }
 
-  case class Config(inputPath: String)
+  case class Config(inputPath: String, dataFlow : Boolean, semanticsFile : String)
```

Comment:
```suggestion
  case class Config(inputPath: String, dataFlow: Boolean, semanticsFile: String)
```

Reply:
I want to run scalafmt in the next PR, because when I did, it also touched files that have nothing to do with this PR, making it hard to review.

Reply:
Fine.